server/server.js

```diff
@@ -3,12 +3,31 @@ const app = express();
 const port = 3000;
 const bodyParser = require('body-parser');
 const { Users, Stocks } = require('./database.js');
+const path = require('path');
 
 app.use(bodyParser.json());
 
 app.use(express.static('public'));
 
+app.get('/api/:ticker', (req, res) => {
+    let ticker = path.basename(req.url);
+    console.log(ticker)
```

Comment:
Just a suggestion, you can delete console.logs after you are done testing with them and before you commit the changes